CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+0.7.1 (to be released)
+----------------------
+
+* It is now possible to log the parameters of requests sent. See **Logging
+  request parameters** in the README for more information.
```

Comment:
I think we can link it with https://github.com/scrapy-plugins/scrapy-zyte-api#logging-request-parameters